src/jsonTypings.ts

```diff
@@ -267,4 +267,39 @@ export namespace JSONObjects {
     export type JSON = Record<string, Random | Obstacle | Wall | Building | Structure | LootTier | Decal>;
 }
 
+export namespace JSONConfig {
+
+    interface ServerConfig {
+        host: string
+        port: number
+        https: boolean
+        webSocketHost: string
+        webSocketPort: number
+        webSocketHttps: boolean
+        webSocketDevAddress: string
+        useWebSocketDevAddress: boolean
+        webSocketRegions: {
+            na: string
+            eu: string
+            as: string
+        }
+        defaultRegion: string
+        keyFile: string
+        certFile: string
+        movementSpeed: number
+        botProtection: boolean
+        stopServerOnGameEnd: boolean
+        debug: {
+            disableMapGeneration: boolean
+            fixedSpawnLocation: number[]
+            disableRedZone: boolean
+            disableStaticFileCache: boolean
+        }
+        diagonalSpeed: number
+    }
+
+    export type JSON = ServerConfig;
```

Comment:
You can export the interface instead of having to create a new type `JSONConfig.JSON` here.

Reply:
I intentionally structured it that way to ensure consistency with the other imports in the file.

```js
export const Objects = readJSON<JSONObjects.JSON>("data/objects.json");
export const Bullets = readJSON<JSONBullets.JSON>("data/bullets.json");
export const Explosions = readJSON<JSONExplosions.JSON>("data/explosions.json");
...
export const Config = readJSON<JSONConfig.JSON>("config.json");
```